tests/bofire/data_models/test_constraints.py

```diff
@@ -133,12 +139,27 @@ def test_constraints_plus():
     [
         (constraints2, 5),
         (constraints4, 5),
+        (constraints5, 5),
+        (constraints6, 5),
     ],
 )
 def test_constraints_call(constraints, num_candidates):
     candidates = inputs.sample(num_candidates, SamplingMethodEnum.UNIFORM)
     returned = constraints(candidates)
-    assert returned.shape == (num_candidates, len(constraints))
+
+    num_rows = 0
```

Comment:
Are we anywhere testing that the call method returns the correct values? We need to urgently refactor our tests :D

Reply:
At least I haven't found them :D But added one testing if the design generation works with batch constraints.